test/csuite/random_directio/main.c

```diff
@@ -438,6 +461,70 @@ schema_operation(WT_SESSION *session, uint32_t threadid, uint64_t id, uint32_t o
     return (ret);
 }
 
+/*
+ * thread_ckpt_run --
+ *     Runner function for the checkpoint thread.
+ */
+static WT_THREAD_RET
+thread_ckpt_run(void *arg)
+{
+    WT_RAND_STATE rnd;
+    WT_SESSION *session;
+    WT_THREAD_DATA *td;
+    uint32_t sleep_time;
+    int i;
+
+    __wt_random_init(&rnd);
+
+    td = (WT_THREAD_DATA *)arg;
+    /*
+     * Keep a separate file with the records we wrote for checking.
+     */
+    testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
+    for (i = 0;; ++i) {
+        sleep_time = __wt_random(&rnd) % MAX_CKPT_INVL;
+        sleep(sleep_time);
+        /*
+         * Since this is the default, send in this string even if running without timestamps.
+         */
```

Comment:
Perhaps a little more explanation here?  I take it that if we say "use_timestamp=true" here and timestamps are off, it will be ignored?  (If so, please add something like that).

Reply:
That was a leftover from pulling it in from timestamp_abort. I have removed the config string and the comment completely as timestamps are not relevant at all in this program.